data_fixtures/migrations/0003_unit_page_copy_updates.py

```diff
@@ -0,0 +1,95 @@
+# Generated by Django 4.2.14 on 2024-07-16 17:30
+
+from django.db import migrations
+
+fixtures = [
+    {
+        "name": "mitpe",
+        "offeror_configuration": {
+            "value_prop": (
+                "MIT Professional Education is a leader in technology and "
+                "engineering education for working professionals pursuing "
+                "career advancement, and organizations seeking to meet modern-day "
+                "challenges by expanding the knowledge and skills of their employees. "
+                "Courses are delivered in a range of formats—in-person (on-campus "
+                "and live virtual), online, and through hybrid approaches—to "
+                "meet the needs of today's learners."
+            ),
+        },
+        "channel_configuration": {
+            "sub_heading": (
```

Comment:
is value_prop and sub_heading different for other offerors? Seems weird that the same copy is stored twice

Reply:
This is odd and stemmed from the unit listing pages being worked alongside the unit page banners.  We should remove this value_prop field and instead derive all this from the channel configuration but I think thats worth its own effort.